### PR TITLE
fix the tutorial of llama to support specify paths

### LINKED ISSUE #403: [Bug]: 运行llama的toturial出错
### Issue Type

Build/Install

### Modules Involved

Documentation/Tutorial/Example

### Current Behavior?

在尝试复现llama实验时，使用原有教程中的llama仓库有问题，因此按照[Issue365](https://github.com/secretflow/spu/issues/365)中的解决方法：
`
python3 -m EasyLM.scripts.convert_checkpoint --load_checkpoint='params::/xxx/open_llama_7b_easylm/open_llama_7b_easylm' --output_file='/xxx/spu/llama-7b.msgpack' --streaming=False
`
结果报错（注意gpt-2的example已经可以跑通）：
Traceback (most recent call last):
  File "/xxx/miniconda3/envs/spu/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/xxx/miniconda3/envs/spu/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/xxx/projects/EasyLM/EasyLM/scripts/convert_checkpoint.py", line 42, in <module>
    mlxu.run(main)
  File "/xxx/miniconda3/envs/spu/lib/python3.8/site-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/xxx/miniconda3/envs/spu/lib/python3.8/site-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
  File "/xxx/projects/EasyLM/EasyLM/scripts/convert_checkpoint.py", line 37, in main
    with mlxu.open_file(FLAGS.output, 'wb') as fout:
  File "/xxx/miniconda3/envs/spu/lib/python3.8/site-packages/absl/flags/_flagvalues.py", line 474, in __getattr__
    raise AttributeError(name)
AttributeError: output

### PR DESCRIPTION
**Pull request by gystar**
# Pull Request

## What problem does this PR solve?
1.examples中llama转换问题
2.examples中llama_split的README没有写清楚模型路径设定
方法
Issue Number: Fixed #403 

## Possible side effects?

- Performance:无。

- Backward compatibility:测试未发现问题。

### CODE REVIEW

**Comment by gystar** [comment]
I have read the CLA Document and I hereby sign the CLA

File: examples/python/ml/flax_llama7b_split/README.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ This example demonstrates how to use SPU to run secure inference on a pre-traine
 
     ```sh
     cd examples/python/ml/flax_llama7b_split
-    python flax_llama7b_split.py --config `pwd`/examples/python/ml/flax_llama7b_split/3pc.json
+    python flax_llama7b_split.py --model_path path-to-flax-llama7b-EasyLM.msgpack  --tokenizer_path path-to-flax-llama7b-dir   --config ./3pc.json
```
Comment (tpppppub) [inline examples/python/ml/flax_llama7b_split/README.md:89]: 上面的 bazel 运行也请改一下

Comment (gystar) [reply examples/python/ml/flax_llama7b_split/README.md:89]: Done

**Review by tpppppub — APPROVED** [review]
LGTM

**Pull request merged**